src/table/ForeignKeyData.php

```diff
@@ -69,7 +69,7 @@ public function renderRefTableName(): string
     public function render(): string
     {
         return str_repeat(' ', 8) . sprintf(
-            '$this->addForeignKey(\'%s\', \'%s\', %s, \'%s\', %s%s%s);',
+            '$this->addForeignKey(\'%s\',' . PHP_EOL . '\'%s\',' . PHP_EOL . ' %s,' . PHP_EOL . ' \'%s\',' . PHP_EOL . ' %s%s%s);',
```

Comment:
It's true that this line is rendered to be usually longer than standard line characters limit in IDEs but I left this like that intentionally so anyone that cares can change it manually in generated migration (because everyone should check the generated file).
Anyway - your PR may be merged but this change is not adding indentation so every line will start at the very edge and I find it worse than just rendering it in one long line. Please correct it.

Reply:
Thanks for your review, I did this PR after case of generating 30+ models and edit them by hand. So i think it will useful for others who will generate more than one model in a row. I'll  change code today. Sorry for my bad english BTW.